openstack_hypervisor/hooks.py

```diff
@@ -29,9 +29,11 @@
 from jinja2 import Environment, FileSystemLoader, Template
 from snaphelpers import Snap
 
+from openstack_hypervisor import utils
```

Comment:
please can we stick to a single style for imports 

from openstack_hypervisor.utils import get_local_ip_by_default_route

also see note about naming of 'utils'